src/dfcx_scrapi/core/agents.py

```diff
@@ -339,7 +340,12 @@ def get_validation_result(
         return val_results_dict
 
 
-    def export_agent(self, agent_id: str, gcs_bucket_uri: str) -> str:
+    def export_agent(
+        self,
+        agent_id: str,
+        gcs_bucket_uri: str,
+        environment: str = None
```

Comment:
I'd recommend renaming `environment` to `environment_name`. This is only because in the Environment class, the variable "environment" is only used for Environment objects. `environment_id` is used for the ID (naturally), and `display_name` is used for environment display name. Since we can't derive that `display_name` references an environment's name from context like we can in the Environments class, I think calling it `environment_name` should suffice.

Reply:
Good point! It makes the code less ambiguous.

Reply:
Let's make this `environment_display_name` to avoid any ambiguity.